L1Trigger/GlobalMuonTrigger/src/L1MuGlobalMuonTrigger.cc

```diff
@@ -174,34 +173,35 @@ void L1MuGlobalMuonTrigger::produce(edm::Event& e, const edm::EventSetup& es) {
   // configure from the event setup
 
   unsigned long long L1MuGMTScalesCacheID = es.get<L1MuGMTScalesRcd>().cacheIdentifier();
+  auto config = m_config.load();
```

Comment:
@makortel , can you review this change? Should I use `m_config.load(std::memory_order_acquire)` instead of `m_config.load()` ?

Reply:
If it should be the former, then I guess this should be changed too?: https://github.com/cms-sw/cmssw/blob/54da0a7414126ad039920833803b0cc512d9a0e4/L1Trigger/GlobalMuonTrigger/src/L1MuGlobalMuonTrigger.cc#L262

Reply:
Honestly I'd just use sequential consistency, i.e. `m_config.load()`, because it is easier to understand and therefore to get correct than the acquire-release semantics. We are hopefully not contending the atomic here.

Reply:
thanks @makortel , I will then change `{m_config,m_db}.load(std::memory_order_acquire)` to `{m_config,m_db}.load()`  which I have added in this PR.